### PR TITLE
feat: Make this bad boy work

### LINKED ISSUE #1: Remove items with no stock remaining
Filter out any catalog item which is out of stock

### LINKED ISSUE #3: Price Display Format
- Integer prices (i.e. £2.00 or £1.00) should display as £2 and £1 respectively
- Prices less than £1.00 should display in pence (e.g. £0.99 should display as 99p)

### PR DESCRIPTION
**Pull request by atoff**
Closes #1, #3 & #4

### CODE REVIEW

**Review by nicolekwli — COMMENTED** [review]
LGTM

**Comment by nicolekwli** [comment]
I take that back, we can't be biased with the J2Os
Screenshot 2022-11-16 at 11 16 12 PM

**Comment by nicolekwli** [comment]
and the sour cream & onion
Screenshot 2022-11-16 at 11 18 48 PM

File: app/page.tsx [hunk 1]
```diff
@@ -16,59 +17,50 @@ const priceFormatter = new Intl.NumberFormat('en-GB', {
...
+  if (price < 100) return `${price}p`
+  if (price % 100 == 0) return `£${price / 100}`
+  return priceFormatter.format(price / 100);
 }
 
-function getItemImage(item: CatalogItem, images: Map<string, CatalogObject>): CatalogImage | null {
-    const imageId = item.imageIds?.[0];
-    if (imageId == null) return null;
-    return images.get(imageId)?.imageData ?? null;
+function getItemImage(item: CatalogItem, images: CatalogObject[]): CatalogImage | null {
```
Comment (JElgar) [inline app/page.tsx:27]: Hmm images seems like the wrong name here now, you can pass on any type

**Review by JElgar — APPROVED** [review]
Lgtm

**Pull request merged**